frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -39,7 +42,7 @@ const buildEChartsLabelOptions = (
       if (dimensionIndex == null || dimensionName == null) {
         return " ";
       }
-      const value = (datum?.value as any)?.[dimensionName];
+      const value = valueGetter((datum?.value as any)?.[dimensionName]);
```

Comment:
Thoughts on maybe just making the function take an additional `settings` parameter instead of having it return a function?

For example
```typescript
const value = valueGetter(datum.value[dimensionName], settings)
```

I don't really have a strong preference for one or the other, so it's fine if we leave it as is